Example/CustomActionControllers/Twitter.swift

```diff
@@ -97,6 +97,7 @@ open class TwitterActionController: ActionController<TwitterCell, ActionData, Tw
         cellSpec = CellSpec.nibFile(nibName: "TwitterCell", bundle: Bundle(for: TwitterCell.self), height: { _ in 56 })
         headerSpec = .cellClass(height: { _ -> CGFloat in return 45 })
         
+        cancelSpec = .nibFile(nibName: "CancelCell", bundle: Bundle(for: CancelCell.self), height: { _ in 46 })
```

Comment:
I wouldn't add this red cancel to this example :) maybe you can create a new example to show how to use the cancelViewSpec. What do you think?